src/test/groovy/org/ods/doc/gen/core/test/wiremock/WiremockManager.groovy

```diff
@@ -47,7 +47,6 @@ class WiremockManager {
                 .wireMockConfig()
                 .usingFilesUnderDirectory(pathToFiles)
                 .dynamicPort()
-                .enableBrowserProxying(true)
```

Comment:
I think this should be false when not recording...

Reply:
As we talked is ok